libs/components/flyout/src/lib/modules/flyout/flyout.service.spec.ts

```diff
@@ -139,13 +138,13 @@ describe('Flyout service', () => {
   }));
 
   it('should remove the host when the user navigates through history if no closed event is fired in 500ms - sanity check', fakeAsync(() => {
-    service.open(SkyFlyoutHostsTestComponent);
+    const instance = service.open(SkyFlyoutHostsTestComponent);
     const dynamicService = TestBed.inject(SkyDynamicComponentService);
     const removeComponentSpy = spyOn(
       dynamicService,
       'removeComponent'
     ).and.callThrough();
-    spyOn(service['host'].instance.messageStream, 'next').and.stub();
+    spyOn(instance.closed, 'emit').and.stub();
```

Comment:
`.and.stub()` is the default behavior when you call `spyOn()` and is only needed to reverse another `.and` function, like `.and.returnValue()` or `.and.callFake()`.